### PR TITLE
Random Battles updates

### PR DESCRIPTION
**Pull request by ACakeWearingAHat**
-Nuzzle will no longer generate a Life Orb on certain pokemon
-Raichu in general has obtained several reworks; it now always gets Water or Grass tera with the accompanying coverage move, it can now run Heavy-Duty Boots and Assault Vest in addition to its other items, and it runs Life Orb generally less often.
-Spidops and Kricketune will now get a Focus Sash in the lead slot
-Several other set updates.

### CODE REVIEW

**Comment by MathyFurret** [comment]
mind also adding the [cleanup Marty proposed](https://github.com/smogon/pokemon-showdown/pull/9238#discussion_r1057849670)?

**Comment by ACakeWearingAHat** [comment]
> mind also adding the [cleanup Marty proposed](https://github.com/smogon/pokemon-showdown/pull/9238#discussion_r1057849670)?

aye, shortly

File: data/random-teams.ts [hunk 1]
```diff
@@ -1190,6 +1190,7 @@ export class RandomTeams {
 		if (species.id === 'toxtricity' && moves.has('shiftgear')) return 'Throat Spray';
 		if (species.id === 'palkia') return 'Lustrous Orb';
 		if (moves.has('substitute') || ability === 'Moody') return 'Leftovers';
+		if (moves.has('stickyweb') && isLead) return 'Focus Sash';
```
Comment (MathyFurret) [inline data/random-teams.ts:1193]: This is fine for now but in the future there might be bulkier webs users who won't need sash

**Review by MathyFurret — APPROVED** [review]

File: data/random-teams.ts (hunk 1 above)
Comment (ACakeWearingAHat) [reply data/random-teams.ts:1193]: yes

**Pull request merged**